app/assets/javascripts/discourse/components/quote-button.js.es6

```diff
@@ -48,8 +48,28 @@ export default Component.extend({
       }
     }
 
+    let opts;
+    for (
+      let element = selectedElement();
+      element;
+      element = element.parentElement
```

Comment:
Do we need to climb the whole DOM tree up to the "root"? Could we just instead bail out as soon as we're out of the body of a post?